src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/index.tsx

```diff
@@ -33,32 +33,32 @@ type Props = AsyncComponent['props'] &
 
 type State = AsyncComponent['state'] & {
   debugFiles: Array<DebugFile> | null;
+  builtinSymbolSources: Array<BuiltinSymbolSource> | null;
 };
 
 class DebugFileDetails extends AsyncComponent<Props, State> {
   getDefaultState() {
     return {
       ...super.getDefaultState(),
       debugFiles: [],
+      builtinSymbolSources: [],
     };
   }
 
   getUplodedDebugFiles(candidates: Candidates) {
-    return candidates.find(candidate => candidate.source === INTERNAL_SOURCE_NAME);
+    return candidates.find(candidate => candidate.source === INTERNAL_SOURCE);
   }
 
   getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
     const {organization, projectId, image} = this.props;
     const {debug_id, candidates = []} = image;
+    const {builtinSymbolSources} = this.state || {};
 
     const uploadedDebugFiles = this.getUplodedDebugFiles(candidates);
+    const endpoints: ReturnType<AsyncComponent['getEndpoints']> = [];
 
-    if (!uploadedDebugFiles) {
-      return [];
-    }
```

Comment:
Is it possible to not have uploaded debug files *and* have builtin symbol sources?

Reply:
yes it is

Reply:
yes it is